apps/upperair/package.json

```diff
@@ -5,7 +5,7 @@
     "start": "meteor run"
   },
   "dependencies": {
-    "@babel/runtime": "^7.14.6",
+    "@babel/runtime": "^7.18.9",
```

Comment:
Did we intend to update the babel runtime for `upperair` in the scorecard branch? If so, should this be done for all the other MATS apps to keep them consistent? It looks like most of them are on v7.14.6 - though there is some variation. E.g. - radar is on v7.17.2.

Reply:
Agreed, if this needed an update it should be done in all apps. I don't think that's a blocker for the pull request, but it should be done soon after.

Reply:
I agree with Molly

Reply:
Captured in #791